### DISCUSSION TITLE
Can I use an existing WPF control in a WinUI application / MAUI?

### DISCUSSION THREAD

**Discussion by anaselhajjaji** [microsoft/WindowsAppSDK#1201] [Q&A] [answered]
Hi, I want to use a WPF legacy control in MAUI / WinUI3, for example [this control](https://www.scichart.com/example/wpf-chart-example-box-plot/). The library developer does not support WinUI. Is there any possibility to use existing WPF controls? thank you.

**Comment by maxkatz6** [comment]
I believe it's technically should be possible to create control handle, aka island, inside of WinUI application, and render there wpf content. Like reversed Xaml Islands. It won't be easy though. 
Hope developers know a bit more straightforward solution.

**Comment by andrewleader** [comment, marked as answer]
Hi @anaselhajjaji, that is not supported yet. Using WPF controls in WinUI 3 is enabled through a feature called "XAML Islands", which is currently planned for a future update after 1.0. You can see the WinUI 3 roadmap here: https://github.com/microsoft/microsoft-ui-xaml/blob/main/docs/roadmap.md#legend
